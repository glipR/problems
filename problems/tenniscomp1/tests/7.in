256
fqnj73opvti23j2 504179966
zgmczh2khy5 157262265
8ilgtccagwuwizys 622239324
ecsf9duyj3 622772972
j5847ix51 868977232
wj38kspui8 605224314
6owljk 860834146
evlphz 597613822
qkryd7uu 299964521
yr6d0x6e7wtnhhxnq 868977232
8e34v 980655177
f3cahwfvaqtzlk 235180
b47blmm0zr 701938378
vapv0b3lcsu63r4ezfru 384696929
6i6fkx9xwylbmj 563458582
bmv9v6r 343743280
z0afprm4jd 350754119
rb2fe1 580282089
3jfdnkmou 860782988
2xbnmx8t422vq8gg71s 520042506
51zhj6xg 114712574
du8v0xt0onr 544795532
1nh3le 331466690
lfqueothft7sq 749229149
ayihl3mxm2k9ogja4l 901905629
7c1u88u 384696929
57m8rbl3yld14pwf 1983062
q60xsacx6vs74l 51891782
ubsdiy22n5 366478032
flbt4 597103075
0w1ek880 771781057
f6yoippbpsajxbh5w31g 187039259
49nwk 35685815
38lrdqmgjxwe 581795686
qhgx9g691ci27 586605851
jbrmd 700550786
qn80v6604ew 990959210
7ew3ne2ryvc3x6 841895850
735ixhgf3udaxisw5 928965492
q3qdy8tib5x0m 385985631
8jjc5l5bs2771j79s 557502322
bv7ul47f3m 554864488
o0vx1bcnaatj8d1f0 581795686
jb60qas1f6ptfpxvad0 969177825
ibt9zw5b9rk67 496699816
pfojroacux 246866835
wxwpa5p4kyqbr 581131722
rvmwcmq6u0x3p4w 436357259
320pn2t9u66mf3s9x6p 78570150
zoefgzp 902448195
lg28db96elm1jqpt 998147862
w6nzovjmbc1u01dfttb 988047762
fmwk0 270899494
p67y582s58on9sjs 882997847
th0u1d2 702323801
i8n1mckn 856829944
gipo7pzqtow1zg 205937267
mthxvgv9c24hb0yt5ke4 383161463
pa3iir6phtyf2 912326254
zcyfmob 567875333
56gm77q9 73383938
bbt4lqgfb 978232864
aticwulgn0f 674498482
4mu9yk4l5 766663612
5iixougkzjzaat 345605522
zyh9jrylj1 841575569
j3v0avkk1nmkv4kqt 359561040
89b1cg41 462373977
6qxjdr0ec3g6yv1b9p 20276479
k8nzihpp73yv1d2bzr 391958895
7su3rizd0sg 495721728
96ne8s 725300935
ia8kvnr5b4b0au 980436131
4femw 613589333
2znxwgw6m0m01 568481164
2qi1hy 64784469
l827favmiqj 846902119
zvl8deuhpowu 925554027
5aixclu0ndlmb1pr66 213280253
5k0uejfrkddtv9gj5 503617107
zmhre6kp8f0ptoj 286724862
cd53jk9iid0foyi36iq 104848356
iwzqhbqvelraldvaj 532227092
ebp1c7f0v6eh 122059859
fcjmopsxys8qmzkf 758776360
irasqytix1 217296583
zlv2s0ffp9cxhjgt 950774648
qmn7ai2wkmptgnsnvg0x 990959210
8go79vn6wg 424467116
iaturp 846732944
cffa1gm1kykks05b 66077710
skx4s7894e6zc9f 856124326
b2wvzvibc3m0f4ft55a 269659974
f997e 726009495
hrevmdsx33o4 165335744
kxxyoh4p33d8wr0von8 503961866
numo4qvz4hcrqh3bmu1 228228482
9g7mld34csp9k 156121151
hv7uhi 352054833
hnc4ytu3z8sndli 35685815
kf9hit 257959407
m6zi090lfmmglf 258374319
jj2jkeoq 553130912
aszuif8d01s3eh 841575569
g4bokvmjp6 856603112
mtxwt2935gyn48f4pdxc 581131722
u3n2ku 91873818
tb4purozqtk5 566453789
6mt67h2fx20c07yvr14g 586534436
tka8poa8u3kdz1ne 66077710
fj6wphz2dcu06sca 743065401
n46hp3 655514745
k6nq0e 509773632
58e3by0jr 648207858
350zi7gcvq3qlrwy01t 990966899
voflieja41wwbx43 151829700
f59v5dklsb 713691327
clfvwu7x67bwqvg 65115356
dq4htrqw2tns 663626335
4km3f 192787536
r6tdcx7lsrexc 57189349
z1to32 347080094
j5tkn 586534436
vka7ansrj4v3 781030252
xbrv6 31203498
tf4zfmm1wtm5frr1r 442892120
ywf2h2ihxg3 20806377
gvmu0bx2tn 905376432
te0rwwldhrrtfo9ec 622772972
e8qz3kttk5y8 570216886
ckpm7ghkacgcu0v 159922782
dokp89dblri 622239324
d9g1249xu 787897733
32nltqogor0pbi 390382071
pz80g3e54i5j 639863745
r3pqmtyxo7 760905752
ezjo0soen1o0gkz 839737699
tqyuwqnlb6me 347895137
3kexz2efc39ha 970490902
mznedx6ns 236785108
kt0aef6xmr 567875333
2tw76dfsisjc 737555445
hx9vcij 120619564
rpg4ayo6jcg0z5x 696094585
iplym0s2hqqpvy9y 120585020
aypxevd6irt6z 370070445
kkxbp7g7nj4e0f 972327035
n6661zpg8b28ynnrlj 827524329
tkbnsc5nw1s2mvj 249161617
31vps9ul 566031848
01ke5n 655514745
7hxd0z 410025240
rtjmvb0sby9 264693233
rm63m8jhp 726009495
z5ixl2g5o 442892120
m3swzx8d71r5upko1mkw 970485944
klrkwf 628847517
m1gh2 66077710
d0yon08wu 580282089
zy26q8ljiy5z 970485944
8xt8poh8kvx 111607761
efzph10304fji0d 440784363
a026bchy3 646969048
no9gtbgict 680369386
3w13orqu5h 990959210
ti7nlz9t 158925802
dqrntcyqwudr41eudx 85869147
vb23d1uqivt8 385985631
dg6w54ca1c50l5p 477721820
n1h56mhl2ngqtvc9m598 970485944
166edl 4525183
q3tw95xfs94o8jkff8 257304942
ujs5v3mt77ilm 157377325
yuri9xrualispd731 948845366
rtnrt 469227967
01wxkokuw6yzi 906927978
4kzdcuesgxy1zr4 440784363
dj5c4f490ikjzm 497347193
z6q6w5 745102489
e8pjdw0uayaxdcs 405513321
ejs94nimimxf5 150489797
2gxoxyh 236785108
8kpsw3q1gxmekdp 526039470
w1l23jczo 766663612
v3zkyeggdhoz 212318347
ha344f8ea3bzq54c 383161463
tnma0ysvcgx0gr3ypx 622239324
bq93y5ozb 303593916
8dzvsb 641593347
tl776x08z0miho59qw8 324405191
ghyjwq69l180k97yg3 401293715
2pondy1fkz 573558349
la5hxokopzv5s 269753709
9ch0ztc8b8snylsx 960033585
ad5u25lfi3skgcv 83962721
4yrggom 563424912
g995t6j 737637294
db5gm2fnoadoebxhb 597613822
rcbk51s4gmsd 359561040
40g4u3jt5pv5jeloq4mb 996939595
va566brekl 836274430
pz9foc3iqs4jjz9gn01r 553422240
wrg7g7g 243171613
zy3bj3m 860834146
i9vfndsh 3971554
agtunmh8rdrfc0qgl 698952468
52adeh0l48w09o 132983736
n5xin6kpszuh01 745062796
0z5441wuryd1 436357259
91qp5gclibozwsyq 274364786
1kiyqgeytt 992539193
gm0xgpd 599362084
7oq9wee7z272 967381543
762vypg 420338693
vdp4xp9o 948644886
uqfl7k18ql 91873818
axesj7p7t 816197306
dzplpsgbr 149482870
cnjdesj81f6ya3 832866366
dnkmham 82060476
l5kqs00vflr 270393000
sanu0y5noidivd9z 187039259
x64yy 586437028
5rmtubbxxnsj9cet 649414118
4su6p1s 789186270
8p9kstf8wb0 446581065
f9gtk9gqd7bw25lr8v 491531462
zevf99eza 639863745
gp9njyjwpkcd 540712026
kdmbi12xk83yz2q 824764857
o5gw0mjhiaaa 605224314
wfa7lh2qqox2 573866223
zp70cxngep6w1g9ozzfl 106580919
7nvxzd 385985631
r59qe8ycknlva 657730366
h1iwjne0dasl0u2mio 640676738
srrl859 192787536
nsiz2uuknhy6qzfsp9o 846902119
0904xjelypi 80143434
fwna6yzthe1czstw4a0 160158053
8pqf40ie 384585615
es0waxp0rf 373391124
cbq4c08zybpwh 648289928
x71u5 838191697
kkxxmed 770615827
fl6uq9ysd 347080094
wn638ri7ds1fh70uqw 495654660
7z3yu03f 909631930
dlawda 545901738
bohllktlimynt 187039259
727nxik5ex6z 755535000
xpmh5 789990706
yz26e 103060570
c84v4v 896356066
apugvfc2ymumql64ltfs 261278710
z3etoo0c4 760905752
