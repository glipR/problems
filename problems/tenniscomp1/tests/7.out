255
fqnj73opvti23j2 defeats zgmczh2khy5
ecsf9duyj3 defeats 8ilgtccagwuwizys
j5847ix51 defeats wj38kspui8
6owljk defeats evlphz
yr6d0x6e7wtnhhxnq defeats qkryd7uu
8e34v defeats f3cahwfvaqtzlk
b47blmm0zr defeats vapv0b3lcsu63r4ezfru
6i6fkx9xwylbmj defeats bmv9v6r
rb2fe1 defeats z0afprm4jd
3jfdnkmou defeats 2xbnmx8t422vq8gg71s
du8v0xt0onr defeats 51zhj6xg
lfqueothft7sq defeats 1nh3le
ayihl3mxm2k9ogja4l defeats 7c1u88u
q60xsacx6vs74l defeats 57m8rbl3yld14pwf
flbt4 defeats ubsdiy22n5
0w1ek880 defeats f6yoippbpsajxbh5w31g
38lrdqmgjxwe defeats 49nwk
jbrmd defeats qhgx9g691ci27
qn80v6604ew defeats 7ew3ne2ryvc3x6
735ixhgf3udaxisw5 defeats q3qdy8tib5x0m
8jjc5l5bs2771j79s defeats bv7ul47f3m
jb60qas1f6ptfpxvad0 defeats o0vx1bcnaatj8d1f0
ibt9zw5b9rk67 defeats pfojroacux
wxwpa5p4kyqbr defeats rvmwcmq6u0x3p4w
zoefgzp defeats 320pn2t9u66mf3s9x6p
lg28db96elm1jqpt defeats w6nzovjmbc1u01dfttb
p67y582s58on9sjs defeats fmwk0
i8n1mckn defeats th0u1d2
mthxvgv9c24hb0yt5ke4 defeats gipo7pzqtow1zg
pa3iir6phtyf2 defeats zcyfmob
bbt4lqgfb defeats 56gm77q9
4mu9yk4l5 defeats aticwulgn0f
zyh9jrylj1 defeats 5iixougkzjzaat
89b1cg41 defeats j3v0avkk1nmkv4kqt
k8nzihpp73yv1d2bzr defeats 6qxjdr0ec3g6yv1b9p
96ne8s defeats 7su3rizd0sg
ia8kvnr5b4b0au defeats 4femw
2znxwgw6m0m01 defeats 2qi1hy
zvl8deuhpowu defeats l827favmiqj
5k0uejfrkddtv9gj5 defeats 5aixclu0ndlmb1pr66
zmhre6kp8f0ptoj defeats cd53jk9iid0foyi36iq
iwzqhbqvelraldvaj defeats ebp1c7f0v6eh
fcjmopsxys8qmzkf defeats irasqytix1
qmn7ai2wkmptgnsnvg0x defeats zlv2s0ffp9cxhjgt
iaturp defeats 8go79vn6wg
skx4s7894e6zc9f defeats cffa1gm1kykks05b
f997e defeats b2wvzvibc3m0f4ft55a
kxxyoh4p33d8wr0von8 defeats hrevmdsx33o4
numo4qvz4hcrqh3bmu1 defeats 9g7mld34csp9k
hv7uhi defeats hnc4ytu3z8sndli
m6zi090lfmmglf defeats kf9hit
aszuif8d01s3eh defeats jj2jkeoq
g4bokvmjp6 defeats mtxwt2935gyn48f4pdxc
tb4purozqtk5 defeats u3n2ku
6mt67h2fx20c07yvr14g defeats tka8poa8u3kdz1ne
fj6wphz2dcu06sca defeats n46hp3
58e3by0jr defeats k6nq0e
350zi7gcvq3qlrwy01t defeats voflieja41wwbx43
f59v5dklsb defeats clfvwu7x67bwqvg
dq4htrqw2tns defeats 4km3f
z1to32 defeats r6tdcx7lsrexc
vka7ansrj4v3 defeats j5tkn
tf4zfmm1wtm5frr1r defeats xbrv6
gvmu0bx2tn defeats ywf2h2ihxg3
te0rwwldhrrtfo9ec defeats e8qz3kttk5y8
dokp89dblri defeats ckpm7ghkacgcu0v
d9g1249xu defeats 32nltqogor0pbi
r3pqmtyxo7 defeats pz80g3e54i5j
ezjo0soen1o0gkz defeats tqyuwqnlb6me
3kexz2efc39ha defeats mznedx6ns
2tw76dfsisjc defeats kt0aef6xmr
rpg4ayo6jcg0z5x defeats hx9vcij
aypxevd6irt6z defeats iplym0s2hqqpvy9y
kkxbp7g7nj4e0f defeats n6661zpg8b28ynnrlj
31vps9ul defeats tkbnsc5nw1s2mvj
01ke5n defeats 7hxd0z
rm63m8jhp defeats rtjmvb0sby9
m3swzx8d71r5upko1mkw defeats z5ixl2g5o
klrkwf defeats m1gh2
zy26q8ljiy5z defeats d0yon08wu
efzph10304fji0d defeats 8xt8poh8kvx
no9gtbgict defeats a026bchy3
3w13orqu5h defeats ti7nlz9t
vb23d1uqivt8 defeats dqrntcyqwudr41eudx
n1h56mhl2ngqtvc9m598 defeats dg6w54ca1c50l5p
q3tw95xfs94o8jkff8 defeats 166edl
yuri9xrualispd731 defeats ujs5v3mt77ilm
01wxkokuw6yzi defeats rtnrt
dj5c4f490ikjzm defeats 4kzdcuesgxy1zr4
z6q6w5 defeats e8pjdw0uayaxdcs
2gxoxyh defeats ejs94nimimxf5
w1l23jczo defeats 8kpsw3q1gxmekdp
ha344f8ea3bzq54c defeats v3zkyeggdhoz
tnma0ysvcgx0gr3ypx defeats bq93y5ozb
8dzvsb defeats tl776x08z0miho59qw8
2pondy1fkz defeats ghyjwq69l180k97yg3
9ch0ztc8b8snylsx defeats la5hxokopzv5s
4yrggom defeats ad5u25lfi3skgcv
g995t6j defeats db5gm2fnoadoebxhb
40g4u3jt5pv5jeloq4mb defeats rcbk51s4gmsd
va566brekl defeats pz9foc3iqs4jjz9gn01r
zy3bj3m defeats wrg7g7g
agtunmh8rdrfc0qgl defeats i9vfndsh
n5xin6kpszuh01 defeats 52adeh0l48w09o
0z5441wuryd1 defeats 91qp5gclibozwsyq
1kiyqgeytt defeats gm0xgpd
7oq9wee7z272 defeats 762vypg
vdp4xp9o defeats uqfl7k18ql
axesj7p7t defeats dzplpsgbr
cnjdesj81f6ya3 defeats dnkmham
l5kqs00vflr defeats sanu0y5noidivd9z
5rmtubbxxnsj9cet defeats x64yy
4su6p1s defeats 8p9kstf8wb0
zevf99eza defeats f9gtk9gqd7bw25lr8v
kdmbi12xk83yz2q defeats gp9njyjwpkcd
o5gw0mjhiaaa defeats wfa7lh2qqox2
7nvxzd defeats zp70cxngep6w1g9ozzfl
r59qe8ycknlva defeats h1iwjne0dasl0u2mio
nsiz2uuknhy6qzfsp9o defeats srrl859
fwna6yzthe1czstw4a0 defeats 0904xjelypi
8pqf40ie defeats es0waxp0rf
x71u5 defeats cbq4c08zybpwh
kkxxmed defeats fl6uq9ysd
7z3yu03f defeats wn638ri7ds1fh70uqw
dlawda defeats bohllktlimynt
xpmh5 defeats 727nxik5ex6z
c84v4v defeats yz26e
z3etoo0c4 defeats apugvfc2ymumql64ltfs
ecsf9duyj3 defeats fqnj73opvti23j2
j5847ix51 defeats 6owljk
8e34v defeats yr6d0x6e7wtnhhxnq
b47blmm0zr defeats 6i6fkx9xwylbmj
3jfdnkmou defeats rb2fe1
lfqueothft7sq defeats du8v0xt0onr
ayihl3mxm2k9ogja4l defeats q60xsacx6vs74l
0w1ek880 defeats flbt4
jbrmd defeats 38lrdqmgjxwe
qn80v6604ew defeats 735ixhgf3udaxisw5
jb60qas1f6ptfpxvad0 defeats 8jjc5l5bs2771j79s
wxwpa5p4kyqbr defeats ibt9zw5b9rk67
lg28db96elm1jqpt defeats zoefgzp
p67y582s58on9sjs defeats i8n1mckn
pa3iir6phtyf2 defeats mthxvgv9c24hb0yt5ke4
bbt4lqgfb defeats 4mu9yk4l5
zyh9jrylj1 defeats 89b1cg41
96ne8s defeats k8nzihpp73yv1d2bzr
ia8kvnr5b4b0au defeats 2znxwgw6m0m01
zvl8deuhpowu defeats 5k0uejfrkddtv9gj5
iwzqhbqvelraldvaj defeats zmhre6kp8f0ptoj
qmn7ai2wkmptgnsnvg0x defeats fcjmopsxys8qmzkf
skx4s7894e6zc9f defeats iaturp
f997e defeats kxxyoh4p33d8wr0von8
hv7uhi defeats numo4qvz4hcrqh3bmu1
aszuif8d01s3eh defeats m6zi090lfmmglf
g4bokvmjp6 defeats tb4purozqtk5
fj6wphz2dcu06sca defeats 6mt67h2fx20c07yvr14g
350zi7gcvq3qlrwy01t defeats 58e3by0jr
f59v5dklsb defeats dq4htrqw2tns
vka7ansrj4v3 defeats z1to32
gvmu0bx2tn defeats tf4zfmm1wtm5frr1r
te0rwwldhrrtfo9ec defeats dokp89dblri
d9g1249xu defeats r3pqmtyxo7
3kexz2efc39ha defeats ezjo0soen1o0gkz
2tw76dfsisjc defeats rpg4ayo6jcg0z5x
kkxbp7g7nj4e0f defeats aypxevd6irt6z
01ke5n defeats 31vps9ul
m3swzx8d71r5upko1mkw defeats rm63m8jhp
zy26q8ljiy5z defeats klrkwf
no9gtbgict defeats efzph10304fji0d
3w13orqu5h defeats vb23d1uqivt8
n1h56mhl2ngqtvc9m598 defeats q3tw95xfs94o8jkff8
yuri9xrualispd731 defeats 01wxkokuw6yzi
z6q6w5 defeats dj5c4f490ikjzm
w1l23jczo defeats 2gxoxyh
tnma0ysvcgx0gr3ypx defeats ha344f8ea3bzq54c
8dzvsb defeats 2pondy1fkz
9ch0ztc8b8snylsx defeats 4yrggom
40g4u3jt5pv5jeloq4mb defeats g995t6j
zy3bj3m defeats va566brekl
n5xin6kpszuh01 defeats agtunmh8rdrfc0qgl
1kiyqgeytt defeats 0z5441wuryd1
7oq9wee7z272 defeats vdp4xp9o
cnjdesj81f6ya3 defeats axesj7p7t
5rmtubbxxnsj9cet defeats l5kqs00vflr
4su6p1s defeats zevf99eza
kdmbi12xk83yz2q defeats o5gw0mjhiaaa
r59qe8ycknlva defeats 7nvxzd
nsiz2uuknhy6qzfsp9o defeats fwna6yzthe1czstw4a0
x71u5 defeats 8pqf40ie
7z3yu03f defeats kkxxmed
xpmh5 defeats dlawda
c84v4v defeats z3etoo0c4
j5847ix51 defeats ecsf9duyj3
8e34v defeats b47blmm0zr
3jfdnkmou defeats lfqueothft7sq
ayihl3mxm2k9ogja4l defeats 0w1ek880
qn80v6604ew defeats jbrmd
jb60qas1f6ptfpxvad0 defeats wxwpa5p4kyqbr
lg28db96elm1jqpt defeats p67y582s58on9sjs
bbt4lqgfb defeats pa3iir6phtyf2
zyh9jrylj1 defeats 96ne8s
ia8kvnr5b4b0au defeats zvl8deuhpowu
qmn7ai2wkmptgnsnvg0x defeats iwzqhbqvelraldvaj
skx4s7894e6zc9f defeats f997e
aszuif8d01s3eh defeats hv7uhi
g4bokvmjp6 defeats fj6wphz2dcu06sca
350zi7gcvq3qlrwy01t defeats f59v5dklsb
gvmu0bx2tn defeats vka7ansrj4v3
d9g1249xu defeats te0rwwldhrrtfo9ec
3kexz2efc39ha defeats 2tw76dfsisjc
kkxbp7g7nj4e0f defeats 01ke5n
m3swzx8d71r5upko1mkw defeats zy26q8ljiy5z
3w13orqu5h defeats no9gtbgict
n1h56mhl2ngqtvc9m598 defeats yuri9xrualispd731
w1l23jczo defeats z6q6w5
8dzvsb defeats tnma0ysvcgx0gr3ypx
40g4u3jt5pv5jeloq4mb defeats 9ch0ztc8b8snylsx
zy3bj3m defeats n5xin6kpszuh01
1kiyqgeytt defeats 7oq9wee7z272
cnjdesj81f6ya3 defeats 5rmtubbxxnsj9cet
kdmbi12xk83yz2q defeats 4su6p1s
nsiz2uuknhy6qzfsp9o defeats r59qe8ycknlva
7z3yu03f defeats x71u5
c84v4v defeats xpmh5
8e34v defeats j5847ix51
ayihl3mxm2k9ogja4l defeats 3jfdnkmou
qn80v6604ew defeats jb60qas1f6ptfpxvad0
lg28db96elm1jqpt defeats bbt4lqgfb
ia8kvnr5b4b0au defeats zyh9jrylj1
qmn7ai2wkmptgnsnvg0x defeats skx4s7894e6zc9f
g4bokvmjp6 defeats aszuif8d01s3eh
350zi7gcvq3qlrwy01t defeats gvmu0bx2tn
3kexz2efc39ha defeats d9g1249xu
kkxbp7g7nj4e0f defeats m3swzx8d71r5upko1mkw
3w13orqu5h defeats n1h56mhl2ngqtvc9m598
w1l23jczo defeats 8dzvsb
40g4u3jt5pv5jeloq4mb defeats zy3bj3m
1kiyqgeytt defeats cnjdesj81f6ya3
nsiz2uuknhy6qzfsp9o defeats kdmbi12xk83yz2q
7z3yu03f defeats c84v4v
8e34v defeats ayihl3mxm2k9ogja4l
lg28db96elm1jqpt defeats qn80v6604ew
qmn7ai2wkmptgnsnvg0x defeats ia8kvnr5b4b0au
350zi7gcvq3qlrwy01t defeats g4bokvmjp6
kkxbp7g7nj4e0f defeats 3kexz2efc39ha
3w13orqu5h defeats w1l23jczo
40g4u3jt5pv5jeloq4mb defeats 1kiyqgeytt
7z3yu03f defeats nsiz2uuknhy6qzfsp9o
lg28db96elm1jqpt defeats 8e34v
350zi7gcvq3qlrwy01t defeats qmn7ai2wkmptgnsnvg0x
3w13orqu5h defeats kkxbp7g7nj4e0f
40g4u3jt5pv5jeloq4mb defeats 7z3yu03f
lg28db96elm1jqpt defeats 350zi7gcvq3qlrwy01t
40g4u3jt5pv5jeloq4mb defeats 3w13orqu5h
lg28db96elm1jqpt defeats 40g4u3jt5pv5jeloq4mb
